packages/dfw/src/dfwcomponents/WorkOrderCell/types.ts

```diff
@@ -0,0 +1,40 @@
+import { Color, IconName } from "@equinor/mad-components";
+import { ViewProps } from "react-native";
+
+export type WorkOrder = {
+    title: string;
+    workOrderId: string;
+    maintenanceType?: string;
+    tagId?: string;
+    equipmentId?: string;
+    activeStatusIds?: string;
+    basicStartDate?: string;
+    basicFinishDate?: string;
+    requiredEnd?: string;
+    workCenterId?: string;
+    functionalLocation?: string;
+};
+
+export type WorkOrderCellProps = {
+    showSymbols?: boolean | "column" | "row";
```

Comment:
This prop can be hard to understand. I would split it into two props.
```suggestion
    showSymbols?: boolean;
    iconDirection?: "column" | "row"
```

You can make `"column"` the default value of iconDirection